benchbuild/projects/gentoo/gentoo.py

```diff
@@ -47,7 +46,7 @@
         path.mkfile_uchroot("/.benchbuild-container")
         benchbuild = find_benchbuild()
         _benchbuild = run.watch(benchbuild)
-        with local.env(BB_VERBOSITY=str(CFG['verbosity'])):
+        with bb.env(BB_VERBOSITY=str(CFG['verbosity'])):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'CFG' is unsubscriptable](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)